kafka-handler/pom.xml

```diff
@@ -114,8 +114,21 @@
       <version>1.7.25</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>kafka-streams-avro-serde</artifactId>
+      <version>4.1.0</version>
```

Comment:
Any reason this is not the latest Confluent version, as of this PR?

Reply:
Changed to 5.1.0